lib/galaxy/tools/parameters/basic.py

```diff
@@ -151,7 +151,7 @@ def value_to_display_text( self, value, app ):
         Convert a value to a text representation suitable for displaying to
         the user
         """
-        if value:
+        if value is not None:
             return unicodify( value )
         return "Not available."
```

Comment:
I disagree that a text parameter with an empty string should return `Not available.` The empty string is a valid setting for a text input, it is misleading to state that it is not available when a tool was originally run with it set.